packages/web-frontend/src/components/PDFExport/A4Page.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
This seems to be the same as the one in lesmis, can we DRY it by moving it to ui-components?

Reply:
Yeah good idea. I also feel `Header` in `A4page` is a bit weird. I've pull it out to address that.